Source/WebCore/rendering/RenderLayer.h

```diff
@@ -120,12 +120,31 @@ enum ShouldApplyRootOffsetToFragments {
     IgnoreRootOffsetForFragments
 };
 
-enum class RequestState {
-    Unknown,
-    DontCare,
-    False,
-    True,
-    Undetermined
+struct RequestState {
```

Comment:
Could you explain in the changelog why these changes were needed?

Reply:
This change was removed.